docs/source/developers/bug_reports.rst

```diff
@@ -18,82 +18,70 @@
 .. _bug-reports:
 
 ********************************
-Report bugs and propose features
+Bug reports and feature requests
 ********************************
 
-Using the software and sharing your experience is a very helpful contribution
-itself. Those who actively develop Arrow need feedback from users on what
-works and what doesn't. Alerting us to unexpected behavior and missing features,
-even if you can't solve the problems yourself, help us understand and prioritize
-work to improve the libraries.
+Arrow relies upon user feedback to identify defects and improvement
+opportunities. All users are encouraged to participate by creating bug reports
+and feature request or commenting on existing issues. Even if you cannot
+contribute solutions to the issues yourself, your feedback helps us understand
+problems and prioritize work to improve the libraries.
 
-We use `JIRA <https://issues.apache.org/jira/projects/ARROW/issues>`_
-to manage our development "todo" list and to maintain changelogs for releases.
-In addition, the project's `Confluence site <https://cwiki.apache.org/confluence/display/ARROW>`_
-has some useful higher-level views of the JIRA issues.
+.. _apache-arrow-jira:
 
-To create a JIRA issue, you'll need to have an account on the ASF JIRA, which
-you can `sign yourself up for <https://issues.apache.org/jira/secure/Signup!default.jspa>`_.
-The JIRA server hosts bugs and issues for multiple Apache projects. The JIRA
-project name for Arrow is "ARROW".
+Apache Arrow Jira
+=================
 
-You don't need any special permissions on JIRA to be able to create issues.
-Once you are more involved in the project and want to do more on JIRA, such as
-assign yourself an issue, you will need "Contributor" permissions on the
-Apache Arrow JIRA. To get this role, ask on the mailing list for a project
-maintainer's help.
+The Arrow project uses `JIRA <https://issues.apache.org/jira/projects/ARROW/issues>`_
+to track issues - both bug reports and feature requests. No account or
+permissions are required to view or search Jira issues. The Jira server
+hosts issue tracking for multiple Apache projects. The Jira project name for
+Arrow is "ARROW".
 
+.. _required-permissions:
 
-.. _jira-tips:
+Required permissions
+++++++++++++++++++++++++++++
 
-Tips for using JIRA
-===================
+Any registered ASF Jira account may create Jira issues in the Apache Arrow
+project without additional permissions. Individuals may
+`create an ASF Jira account here <https://issues.apache.org/jira/secure/Signup!default.jspa>`_.
+Self-assigning issues to work requires
+the "Contributor" permission. This permission is set by project maintainers,
+and may be requested by emailing the dev mailing list.
```

Comment:
Should we add the full email address? As a new user reporting an issue it might not be clear where to find the dev mailing list.
```suggestion
and may be requested by emailing the `dev@arrow.apache.org <https://mail-archives.apache.org/mod_mbox/arrow-dev/>`_ mailing list.
```

Reply:
Good idea! I changed this to point to the mailing list section of the communication page, just because I'm not sure of the mechanics of a non-subscriber emailing the dev@ list (will it go through? will they see any reply?).